OneWelcomeExampleApp.xcodeproj/project.pbxproj

```diff
@@ -127,6 +127,8 @@
 		8D5B76CE2880417900876B0C /* UIColorExtension.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8D5B76CD2880417900876B0C /* UIColorExtension.swift */; };
```

Comment:
please also resolve the building issue:

`error build: Embedded binary's bundle identifier is not prefixed with the parent app's bundle identifier.
`

Reply:
Done I guess.